routers/web/auth/oauth.go

```diff
@@ -326,10 +325,29 @@ func getOAuthGroupsForUser(ctx go_context.Context, user *user_model.User) ([]str
 	return groups, nil
 }
 
+func parseBasicAuth(ctx *context.Context) (username, password string, err error) {
+	authHeader := ctx.Req.Header.Get("Authorization")
+	if authType, authData, ok := strings.Cut(authHeader, " "); ok && authType == "Basic" {
```

Comment:
I believe auth scheme "Basic" should be compare case insensitively

> "Note that both scheme and parameter names are matched case-insensitively."
> https://www.rfc-editor.org/rfc/rfc7617.html#section-2

Reply:
Good catch. @slingamn was that a deliberate change or we just missed it in reviewing?

Reply:
Wasn't intentional :-) I just refactored the original implementation (#6293) which was case-sensitive. So there's no behavior change, but I agree that a case-insensitive comparison would be more correct.